global-intuitive.postman_collection.json

```diff
@@ -0,0 +1,223 @@
+{
+	"variables": [],
+	"info": {
+		"name": "Global Intuitive",
+		"_postman_id": "a5f45c87-bb07-1668-4f58-eeacc29252bb",
+		"description": "Fast and efficient global REST API for capturing validated addresses. Allows users to search and format their address in just a few keystrokes. ",
+		"schema": "https://schema.getpostman.com/json/collection/v2.0.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "Search",
+			"request": {
+				"url": {
+					"raw": "https://api.edq.com/capture/address/v2/search?query=<search query>&country=<3 letter country ISO code>&take=<number of results>",
+					"protocol": "https",
+					"host": [
+						"api",
+						"edq",
+						"com"
+					],
+					"path": [
+						"capture",
+						"address",
+						"v2",
+						"search"
+					],
+					"query": [
+						{
+							"key": "query",
+							"value": "<search query>",
+							"equals": true,
+							"description": "Address to search on - the input from the user so far"
+						},
+						{
+							"key": "country",
+							"value": "<3 letter country ISO code>",
+							"equals": true,
+							"description": "Must be a licensed country. See documentation for full list of supported ISO codes."
+						},
+						{
+							"key": "take",
+							"value": "<number of results>",
+							"equals": true,
+							"description": "Optional parameter, default is 7."
+						}
+					],
+					"variable": []
+				},
+				"method": "GET",
+				"header": [
+					{
+						"key": "Auth-Token",
+						"value": "{{Auth-Token}}",
+						"description": ""
+					}
+				],
+				"body": {},
+				"description": "Search for an address. Returns an object containing suggested addresses based on the included parameters."
+			},
+			"response": [
```

Comment:
Liking the presence of examples - worth considering adding examples to the other collections at some point - just don't put real auth tokens in..